core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -579,7 +625,10 @@ case object ModelInsights {
                 case _ => Map.empty[String, Double]
               },
               contribution =
-                keptIndex.map(i => contributions.map(_.applyOrElse(i, (_: Int) => 0.0))).getOrElse(Seq.empty),
+                if (LRStandardization && sparkFtrContrib.nonEmpty) {
+                  sparkFtrContrib.updated(0, sparkFtrContrib.head * featureStd / labelStd)
```

Comment:
why head? sparkFtrContrib.head is this for binary only? - I dont think as written it will effect binary only

Reply:
i'm descaling the first coeff in `sparkFtrContrib` for each (derived) feature. This works for binary logistic regression & linear regression, but will break/ won't fully descale multi-nomial logistic regression for multi-class classification.

Reply:
shouldnt you scale the entire vector?